src/backend/aspen/api/views/phylo_trees.py

```diff
@@ -0,0 +1,51 @@
+import sentry_sdk
+import sqlalchemy as sa
+from fastapi import APIRouter, Depends
+from sqlalchemy.ext.asyncio import AsyncSession
+from sqlalchemy.orm import aliased, joinedload
+from sqlalchemy.sql.selectable import Select
+
+from aspen.api.auth import get_auth_user
+from aspen.api.deps import get_db
+from aspen.api.error import http_exceptions as ex
+from aspen.api.schemas.phylo_trees import PhyloTreeRequest, PhyloTreeResponse
+from aspen.database.models import Group, PhyloRun, PhyloTree, User
+
+router = APIRouter()
+
+
+@router.put("/")
+async def update_phylo_tree(
```

Comment:
The rest of our CRUD endpoints for phylo trees are actually set up at the phylo *run* level -- to be able to list in-progress phylo trees, we decided the best way to do that was to get a list of phylo runs (since trees don't get created till the end of a run). And once we made that decision, it didn't make sense to delete a tree without deleting its associated phylo run either.

This update endpoint is at the *tree* level, which has some tradeoffs:
- once a phylo run is complete, I can see how we *might* want to consider the run to be immutable, while the tree metadata can be updated... so from that angle only allowing the trees to be updated makes sense
- but this is a different API interface than the rest of the UI has been counting on, so the UI will need to keep track of the phylo tree ID in our phylo runs response to be able to update the right tree.

There are probably more pros/cons than I've had a chance to think through, but I'm not sure what your take on this is.

Reply:
hmm i'll move this over to phyloruns since the phylotree/phylorun models are closely entwined at the moment. Should we allow users to update started PhyloRuns or would that mess up the tree building pipeline?